app/konnect/org-management/okta-idp.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Set Up Okta IdP Authentication
+no_version: true
+badge: enterprise
+---
+
+As an alternative to Konnect Cloud’s native basic authentication, you can set up
+org access through Okta. This way, your users can log in to Konnect Cloud
+using their Okta credentials, and without needing a separate login.
+
+You can't mix authenticators in Konnect Cloud. With Okta IdP
+authentication enabled, all non-admin Konnect users will subsequently log in
+through Okta. Only Konnect admins can continue to log in with basic
```

Comment:
Technically, only the org owner will be able to login using Konnect authentication. i.e. the person that signed up for the account originally.